builtin_json.go

```diff
@@ -20,6 +21,9 @@ func (r *Runtime) builtinJSON_parse(call FunctionCall) Value {
 	d := json.NewDecoder(strings.NewReader(call.Argument(0).toString().String()))
 
 	value, err := r.builtinJSON_decodeValue(d)
+	if errors.Is(err, io.EOF) {
+		err = fmt.Errorf("Unexpected end of JSON input (%w)", err)
```

Comment:
It doesn't really make sense to create an error, considering we're only interested in its string value. Also, Go convention is not to begin an error message with a capital letter, so some IDEs would highlight this. It's better to do something like:
```go
panic(r.newError(r.global.SyntaxError, fmt.Sprintf("Unexpected end of JSON input (%v)", err)))
```

Reply:
Sure, that makes sense. I've adjusted :+1: